frontend/workflows/k8s/src/k8s-dash-search.tsx

```diff
@@ -14,20 +14,21 @@ import { useDataLayout } from "@clutch-sh/data-layout";
 import styled from "@emotion/styled";
 import { yupResolver } from "@hookform/resolvers/yup";
 import SearchIcon from "@mui/icons-material/Search";
+import type { Theme } from "@mui/material";
 import * as yup from "yup";
 
-const Container = styled.div({
-  margin: "32px 0",
-});
+const Container = styled.div(({ theme }: { theme: Theme }) => ({
+  margin: theme.spacing(theme.clutch.spacing.lg, theme.clutch.spacing.none),
```

Comment:
No idea but I wonder if we can override how theme.spacing works and then just be able to provide the levels, like:

```
theme.spacing('lg', 'none');
...
theme.spacing('lg', '16px');
```

Not anything that needs to happen here and might be more trouble than it's worth but could be nice for the future

Reply:
The example provided in the documentation seems to indicate that this is possible when using string values:
- https://mui.com/material-ui/customization/spacing/#multiple-arity

Reply:
It seems that we could add our own spacing function:

- https://github.com/mui/material-ui/blob/429b48a408c8388e2a5cf629874eb467abe4af35/packages/mui-system/src/createTheme/createSpacing.ts#L28 

Seemingly, there's no docs describing this functionality but this may enable the wanted functionality.

Reply:
I believe there's an [option](https://mui.com/material-ui/customization/spacing/#custom-spacing) for intercepting string values passed to `theme.spacing` and using the values you defined previously to make a substitution
```
const theme = createTheme({
  spacing: (...args) => // Parse args, substitute strings with clutch.spacing values
});
```